src/__tests__/TuxArchitectStores-test.js

```diff
@@ -99,31 +99,35 @@ describe('architect', function () {
   });
 
   it('should allow a store to output multiple outputs', function () {
```

Comment:
Change test string expectation to read, 'should allow a store to have multiple outputs'.